index.html

```diff
@@ -196,5 +196,39 @@ <h2>About me</h2>
     </div>
   </section>
 
-  
+ <!-- contact and form section -->
+
+ <section class="contact-section">
+    <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="contact-form" action="https://formspree.io/f/xwkjrkqw" method="post" id="form-element">
+      <ul>
```

Comment:
- [ ] Please avoid using the `ul` inside the form. You can still use the form and with the help of `flexbox` you can easily attain the structure. As I noticed that you used the `flexbox` on your `ul` but you can optimize it more and make your code semantically more correct which makes your website more accessible. Whereas please note that `ul` and `li` are just meant for listing purposes so use it wisely 😄 . If you do the changes your code looks more leaner and easy to understand.